internal/evidence/reactor.go

```diff
@@ -35,7 +35,7 @@ const (
 func GetChannelDescriptor() *p2p.ChannelDescriptor {
 	return &p2p.ChannelDescriptor{
 		ID:                  EvidenceChannel,
-		MessageType:         new(tmproto.EvidenceList),
+		MessageType:         new(tmproto.Evidence),
```

Comment:
this might be protocol impacting and would require a spec change?

Reply:
Yes I think you're right

Reply:
In light of this, it may be worth adding a changelog for this too.

Reply:
It will be added to the changelog as part of the following PR: https://github.com/tendermint/tendermint/pull/7765